.github/workflows/earthly-build.yaml

```diff
@@ -42,10 +42,17 @@ jobs:
         run: |
           earthly --ci --output +sources
```

Comment:
you can add before here to tweak image name locally in Earthfile (it is a hack...)

```
          sed -i s/osrf/space-ros/g Earthfile # need to replace org name to push successfully later using Earthly
```

Reply:
@asimonov see below - I added a target specifically for pushing images. And now we save locally built images with the branch tag by default (or just HEAD if you happen to be on a random commit...).

Still working out how to test this, though.